### PR TITLE
docker-compose changes

### PR DESCRIPTION
**Pull request by mry-usds**
This PR fixes docker-compose

## Changes
- trapped for no transports
- removed address from organizations-local.yml

## Checklist
- [X] Tested locally?
- [ ] Added new dependencies?
- [ ] Updated the docs?
- [ ] Added a test?
- [X] Did you check for sensitive data, and remove any?
- [ ] Are additional approvals needed for this change?
- [ ] Are there potential vulnerabilities or licensing issues with any new dependencies introduced?

## Fixes 

- #issue

## To Be Done

- #issue

### CODE REVIEW

**Review by RickHawesUSDS — APPROVED** [review]
LGTM

File: prime-router/src/main/kotlin/azure/SendFunction.kt [hunk 1]
```diff
@@ -34,15 +34,17 @@ class SendFunction {
...
+                        when (it) {
+                            is SFTPTransportType -> SftpTransport().send(service.fullName, it, header, content)
+                            else -> true
+                        }
+                    }.reduce { acc, s -> acc && s }
+
+                    if (success) {
+                        context.logger.info("Sent report: ${header.task.reportId} to ${service.fullName}")
                     }
-                }.reduce { acc, s -> acc && s }
```
Comment (RickHawesUSDS) [inline prime-router/src/main/kotlin/azure/SendFunction.kt:42]: The .map function appears to throw the exception, but also call the following reduce. So, I'm seeing logs that have both an SFTP exception and the map-reduce exception. Here's my non-functional sytle suggestion that avoids the map function.
```
                if (service.transforms.isNotEmpty()) {
                    var success: Boolean = true
                    service.transports.forEach {
                        success = success && when (it) {
                            is SFTPTransportType -> SftpTransport().send(service.fullName, it, header, content)
                            else -> true
                        }
                    }
                    if (success) {
                        context.logger.info("Sent report: ${header.task.reportId} to ${service.fullName}")
                    }
                }
```

**Review by RickHawesUSDS — CHANGES_REQUESTED** [review]
So, where is the docker-compose.yml change?

File: prime-router/src/main/kotlin/azure/SendFunction.kt (hunk 1 above)
Comment (mry-usds) [reply prime-router/src/main/kotlin/azure/SendFunction.kt:42]: Inserted...

File: prime-router/src/main/kotlin/azure/SendFunction.kt [hunk 2]
```diff
@@ -34,15 +34,17 @@ class SendFunction {
 
                 val content = workflowEngine.readBody(header)
 
-                val success: Boolean = service.transports.map {
-                    when (it) {
-                        is SFTPTransportType -> SftpTransport().send(service.fullName, it, header, content)
-                        else -> true
+                if (service.transforms.isNotEmpty()) {
+                    var success: Boolean = true
```
Comment (RickHawesUSDS) [inline prime-router/src/main/kotlin/azure/SendFunction.kt:38]: My suggestion had a typo. Replace `transforms` with `transports`

**Review by RickHawesUSDS — CHANGES_REQUESTED** [review]

File: prime-router/src/main/kotlin/azure/SendFunction.kt (hunk 2 above)
Comment (mry-usds) [reply prime-router/src/main/kotlin/azure/SendFunction.kt:38]: updated...

**Review by RickHawesUSDS — APPROVED** [review]

**Pull request merged**